### PR TITLE
Fix images from addons frontend blog utils

### LINKED ISSUE #414: Firefox Logo fails to be displayed on Add-ons Blog
STR: 
1. Launch AMO Homepage https://addons.allizom.org/en-US/firefox/ 
2. Access Firefox Add-ons Blog https://addons.allizom.org/blog/  
3. Firefox Logo it’s not displayed on the Add-ons Blog header  

Expected: 
Firefox Logo should be displayed on the Add-ons Blog 

Actual: 
Firefox Logo fails to be displayed on the Add-ons Blog Header  

Notes: When clicking on the header the page redirects to the AMO Homepage but the logo it’s not displayed  
Screenshot 2022-11-16 at 11 35 35

### PR DESCRIPTION
**Pull request by diox**
Fixes #414

### CODE REVIEW

File: bin/build-styles [hunk 1]
```diff
@@ -10,7 +13,36 @@ const inputFile = './src/assets/css/styles.scss';
...
+  // addons-frontend-blog-utils style.css has paths to images, we want to copy
+  // those to the right directory, rewrite the stylesheet, then the rewritten
+  // file can be imported from ours.
+  const options = {
+    url: 'copy',
+    basePath: path.resolve('node_modules/addons-frontend-blog-utils'),
+    // assetsPath is supposed to be relative to `to` or absolute, but it
+    // doesn't seem to work correctly with absolute paths or with '../img', so
+    // we set it to be empty and allow the images to live inside 'css/'.
+    assetsPath: '',
```
Comment (diox) [inline bin/build-styles:33]: This isn't great, but it works... I may be doing things the wrong way, but I couldn't get `assetsPath` to work like it should with absolute paths (or relative paths with `../`)

Comment (diox) [reply bin/build-styles]: Found a fix in https://github.com/mozilla/addons-blog/pull/415/commits/7c689df9847427910a5e5918dd78d9dcec1441fb

File: bin/build-styles [hunk 2]
```diff
@@ -10,7 +13,39 @@ const inputFile = './src/assets/css/styles.scss';
...
+  // those to the right directory, rewrite the stylesheet, then the rewritten
+  // file can be imported from ours.
+  const options = {
+    url: 'copy',
+    basePath: path.resolve('node_modules/addons-frontend-blog-utils'),
+    assetsPath,
+    useHash: true,
+  };
+
+  await fs.mkdir(assetsPath, { recursive: true });
```
Comment (bobsilverberg) [inline bin/build-styles:35]: Just curious about why you're using `await fs.mkdir` instead of `fs.mkdirSync`?

**Review by bobsilverberg — APPROVED** [review]
Looks good, just a curious question.

r+wc

File: bin/build-styles (hunk 2 above)
Comment (diox) [reply bin/build-styles:35]: Consistency with the rest of the scripts in addons-blog that are written that way

**Pull request merged**